src/bp-moderation/classes/class-bp-moderation.php

```diff
@@ -1128,15 +1128,16 @@ public function save() {
 			} else {
 				$this->count += 1;
 			}
-			if ( ! empty( $threshold ) ) {
-				if ( $this->count >= $threshold && empty( $this->hide_sitewide ) ) {
-					$this->hide_sitewide = 1;
-					$auto_hide           = true;
-				}
-				if ( BP_Moderation_Members::$moderation_type === $this->item_type && $this->count_report >= $user_threshold && empty( $this->hide_sitewide ) ) {
-					$this->hide_sitewide = 1;
-					$auto_hide           = true;
-				}
+
+			if ( ! empty( $threshold ) && $this->count >= $threshold && ( $this->user_report !== 1) && empty( $this->hide_sitewide ) ) {
```

Comment:
```suggestion
			if (
				! empty( $threshold ) &&
				$this->count >= $threshold &&
				1 !== $this->user_report &&
				empty( $this->hide_sitewide )
			) {
```

Reply:
@rajinsharwar2 
Why need to add separate bracket for this one `( $this->user_report !== 1)` if we have all and condition?
Also make sure to add yoda condition